doc/appendices/administration.rst

```diff
@@ -0,0 +1,120 @@
+.. Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+.. See the doc/COPYRIGHT file for details.
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _administration:
+
+**************
+Administration
+**************
+
+This appendix includes administrative details describing the roles and
+responsibilities of participants.
+
+
+Community standards
+===================
+
+Be respectful. Communicate openly. Resolve disagreements peacefully (or even
+agree peacefully to postpone their resolution).
+
+
+.. _roles:
+
+Roles
+=====
+
+The roles of the Celeritas code base are related but not identical to the roles
+set out in the SciDAC project management plan (PMP). These roles should be
+reflected in the different teams and/or access restrictions in GitHub (TODO:
+they currently are not).
+
+Code lead
+   The code lead (currently `@sethrj`_) is responsible for steering the
+   technical implementation of the codebase to meet the long-term project
+   goals. This should be the same person as the "core capability lead"
+   described in the PMP. One key responsibility of the code lead is to set up
+   milestones for releases and execute the release process described below.
+
+Maintainer
+   Maintainers should be familiar with most or all parts of the codebase and
+   are responsible for merging pull requests and closing issues.
+
+Core team member
+   The "core team" are those currently funded to work on Celeritas. Core team
+   members are responsible for reviewing pull requests in accordance with the
+   :ref:`contributing guidelines <contributing>`. They should regularly
+   contribute new code or publish new results using Celeritas.
+
+Contributor
+   Anyone can submit a pull request that conforms to the contribution
+   guidelines! Contributors are recognized by including their handle and pull
+   requests in release notes.
+
+.. _@sethrj: https://github.com/sethrj
+
+
+Release process
+===============
+
+Celeritas uses `Semantic Versioning`_ to enumerate releases. During its initial
+development phase, ``0.x.0`` is a major release and ``0.x.z`` is a patch
+release. When Celeritas is declared stable, ``x.0.0`` is a major release,
+``x.y.0`` is a minor release, and ``x.y.z`` is a patch release.
+
+Major and minor releases (including 0.x.0 development releases) must have a
+milestone in the git issue tracker with a list of issues that can be assigned.
+Only major releases can deprecate or remove features and change
+:ref:`public-facing APIs <api>`. Both major and minor releases should include
+notable improvements to the code.
+
+Patch releases can be created at any time but should typically include at least
+one critical bug fix or several substantial fixes. Patch releases should focus
+almost exclusively on fixes and should generally not include new features or
+other major code changes.
+
+.. _Semantic Versioning: https://semver.org
+
+
+Releases can be created from the master branch (major, minor, patch) or a
```

Comment:
Should we (start) migrate toward using 'main' to refer to the development branch (see also https://github.com/github/renaming)
```suggestion
Releases can be created from the main branch (major, minor, patch) or a
```